administrator/language/de-DE/plg_multifactorauth_fixed.ini

```
@@ -0,0 +1,18 @@
+; Joomla! German Translation
+; (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+; (C) Translation 2008 - 2022 J!German <https://www.jgerman.de>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_MULTIFACTORAUTH_FIXED="Multi-Faktor-Authentifizierung - Festgelegter Code"
+PLG_MULTIFACTORAUTH_FIXED_ERR_EMPTYCODE="Das Feld „Festgelegter Code“ darf nicht leer sein."
+PLG_MULTIFACTORAUTH_FIXED_LBL_DEFAULTTITLE="Festgelegter Code"
+PLG_MULTIFACTORAUTH_FIXED_LBL_DISPLAYEDAS="Festgelegter Code"
+PLG_MULTIFACTORAUTH_FIXED_LBL_LABEL="Festgelegter Code"
+PLG_MULTIFACTORAUTH_FIXED_LBL_PLACEHOLDER="Festgelegten Code eingeben"
+PLG_MULTIFACTORAUTH_FIXED_LBL_POSTMESSAGE="<p class=\"alert alert-info\">Die Nachrichten, über und unter dem Codefeld, können durch Überschreiben der Sprachstrings <code>PLG_MULTIFACTORAUTH_FIXED_LBL_PREMESSAGE</code> und <code>PLG_MULTIFACTORAUTH_FIXED_LBL_POSTMESSAGE</code> angepasst werden.</p>"
+PLG_MULTIFACTORAUTH_FIXED_LBL_PREMESSAGE="<p>Dies ist ein Demonstrations-Plugin für Multi-Faktor-Authentifizierung für Joomla. Es muss der festgelegte Code eingeben werden, welcher bei der Aktivierung der Multi-Faktor-Authentifizierung für diesen Benutzer konfiguriert wurde. Es funktioniert effektiv wie ein zweites Passwort.</p>"
+PLG_MULTIFACTORAUTH_FIXED_LBL_SETUP_POSTMESSAGE="<p class=\"alert alert-info\">Die Nachrichten, oberhalb und unterhalb des Setup-Bereichs, können durch Überschreiben der Sprachstrings <code>PLG_MULTIFACTORAUTH_FIXED_LBL_SETUP_PREMESSAGE</code> und <code>PLG_MULTIFACTORAUTH_FIXED_LBL_SETUP_POSTMESSAGE</code> angepasst werden.</p>"
+PLG_MULTIFACTORAUTH_FIXED_LBL_SETUP_PREMESSAGE="<p>Unten einen festen Code eingeben. Dieser feste Code muss nach dem Einloggen eingegeben werden, bevor die Website aufgerufen werden kann.</p>"
+PLG_MULTIFACTORAUTH_FIXED_LBL_SHORTINFO="Eigenen festen, vordefinierten Code wählen. <strong>Nur zu Demonstrationszwecken</strong>."
+PLG_MULTIFACTORAUTH_FIXED_XML_DESCRIPTION="Ein Demonstrations-Plugin für die mehrstufige Authentifizierung, das einen festen Code (ein &bdquo;zweites Passwort&rdquo;) verwendet. <strong>Nicht auf Live-Sites verwenden, es ist nicht sicher.</strong> Dieses Plugin soll nur als Beispiel für Entwickler dienen, die daran interessiert sind, ihre eigenen Plugins zu erstellen."
```

```suggestion
PLG_MULTIFACTORAUTH_FIXED_XML_DESCRIPTION="Ein Demonstrations-Plugin für die Multi-Faktor-Authentifizierung, das einen festen Code (ein &bdquo;zweites Passwort&rdquo;) verwendet. <strong>Nicht auf Liveseiten verwenden, da es nicht sicher ist.</strong> Dieses Plugin soll nur als Beispiel für Entwickler dienen, die daran interessiert sind, ihre eigenen Plugins zu erstellen."
```

Liveseite:
https://github.com/joomlagerman/joomla/blob/558352aa9868f09e4d7a178e40fa8b1e72d11efb/administrator/language/de-DE/com_config.ini#L71